.github/workflows/version-patch.yml

```
@@ -0,0 +1,154 @@
+name: Version patch
+run-name: patch - ${{ inputs.tag }} by @${{ github.actor }}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        required: true
+        description: 'Specify the tag for this release'
+        default: 'v11.x'
+        type: string
+
+      existing-tag:
+        required: true
+        description:
+          'Specify the pre-existing tag of the release you want to publish a
+          patch for. This will most likely be the latest release tag. For
+          instance, to release v11.80.2, the existing tag would be v11.80.1. To
+          release v11.81.1, the existing tag would be v11.81.0. To find the
+          previous release, view the tag list
+          https://github.com/carbon-design-system/carbon/tags'
+        default: 'v11.x'
+        type: string
+
+      commit-1:
+        required: true
+        description:
+          'Specify the SHA of a commit that should be included in this release.'
+        default: '########################################'
+        type: string
+
+      commit-2:
+        required: false
+        description:
+          'Optionally specify a 2nd commit SHA to be included in the release'
+        default: '########################################'
+        type: string
+
+      commit-3:
+        required: false
+        description:
+          'Optionally specify a 3rd commit SHA to be included in the release'
+        default: '########################################'
+        type: string
+
+      commit-4:
+        required: false
+        description:
+          'Optionally specify a 4th commit SHA to be included in the release'
+        default: '########################################'
+        type: string
+
+      commit-5:
+        required: false
+        description:
+          'Optionally specify a 5th commit SHA to be included in the release.'
+        default: '########################################'
+        type: string
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    if:
+      ${{ github.repository == 'carbon-design-system/carbon' &&
+      github.event.inputs.tag != 'v11.x' && github.event.inputs.existing-tag !=
+      'v11.x' }}
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: '0'
+          fetch-tags: true
+          ref: ${{ github.event.inputs.existing-tag }}
+      - name: Use Node.js version from .nvmrc
+        uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4.3.0
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Create new release branch
+        run: |
+          git checkout -b ${{ github.event.inputs.tag }}
```

Should this be changed given the the code in the `Create release commit` step? Or are they intentionally different?

```suggestion
          git checkout -b release/${{ github.event.inputs.tag }}
```

`Create release commit` code:

https://github.com/carbon-design-system/carbon/blob/cfb0c1ac5067c97da4188b9c05e0b317bdbaf804/.github/workflows/version-patch.yml#L134